src/main/resources/db/migration/0250/V266__MigrateSpeciesGrowthForm.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO public.species_growth_forms (species_id, growth_form_id)
+SELECT id, growth_form_id
+FROM public.species
+WHERE growth_form_id IS NOT null
+ON CONFLICT DO NOTHING;
+
+ALTER TABLE public.species DROP COLUMN growth_form_id;
```

Comment:
May as well combine these into one migration since we're not doing a phased backward-compatible rollout for this change.